README.md

```diff
@@ -1,7 +1,7 @@
 [![HoundCI](https://img.shields.io/badge/reviewed%20by-Hound-%23a873d1.svg)](https://houndci.com)
-[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/badge.svg?branch=ft-setRoles-187355108)](https://coveralls.io/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn?branch=ft-setRoles-187355108)
 
-[![CircleCI](https://circleci.com/gh/atlp-rwanda/e-commerce-ones-and-zeroes-bn/tree/develop.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/?branch=develop)
+[![CircleCI](https://circleci.com/gh/atlp-rwanda/e-commerce-ones-and-zeroes-bn/tree/develop.svg?style=svg)](https://app.circleci.com/pipelines/github/atlp-rwanda/e-commerce-ones-and-zeroes-bn/?branch=ft-setRoles-187355108)
```

Comment:
Please do not change these, we need to see the coverage and CI statuses on `develop` branch